commands/publish/lib/get-unpublished-packages.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const fetch = require("npm-registry-fetch");
+const log = require("npmlog");
+const pReduce = require("p-reduce");
+
+module.exports = getUnpublishedPackages;
+
+// Only the abbreviated package metadata is needed to
+// determine which versions have been published. This
+// saves transfer time for packages with a lot of
+// history.
+const registryOptions = {
+  Accept: "application/vnd.npm.install-v1+json",
```

Comment:
I think we should ultimately be using [`pacote.packument()`](https://github.com/zkat/pacote#--pacotepackumentspec-opts) instead of calling `npm-registry-fetch` directly. It caches stuff, as well as handling this "corgi doc" header.

Reply:
Of course I left out the link I was trying to add in the request for changes: https://github.com/lerna/lerna/blob/ad649bf965a6452d5f4aad8026f4ed62a548cada/commands/add/index.js#L165